cmd/manager/main.go

```diff
@@ -41,9 +42,42 @@ import (
 func main() {
 	klog.InitFlags(nil)
 
-	watchNamespace := flag.String("namespace", "", "Namespace that the controller watches to reconcile machine-api objects. If unspecified, the controller watches for machine-api objects across all namespaces.")
-	metricsAddr := flag.String("metrics-addr", ":8080", "The address the metric endpoint binds to.")
-	healthAddr := flag.String("health-addr", ":9440", "The address for health checking.")
+	watchNamespace := flag.String(
+		"namespace",
+		"",
+		"Namespace that the controller watches to reconcile machine-api objects. If unspecified, the controller watches for machine-api objects across all namespaces.",
+	)
+
+	metricsAddr := flag.String(
+		"metrics-addr",
+		":8080",
+		"The address the metric endpoint binds to.",
+	)
+
+	healthAddr := flag.String(
+		"health-addr",
+		":9440",
+		"The address for health checking.",
+	)
+
+	leaderElectResourceNamespace := flag.String(
+		"leader-elect-resource-namespace",
+		"",
+		"The namespace of resource object that is used for locking during leader election. If unspecified and running in cluster, defaults to the service account namespace for the controller. Required for leader-election outside of a cluster.",
+	)
+
+	leaderElect := flag.Bool(
+		"leader-elect",
+		false,
```

Comment:
don't we want this to be true?

Reply:
It will be set on true in in-cluster environment (https://github.com/openshift/machine-api-operator/commit/f0929b94323d845762ed045c0e8859948cca4fd3), but for local development this feature has no value, and having this set on false is just simpler to use.